README.md

```diff
@@ -59,6 +59,11 @@ services networked together to make up all the components of the Lumigator appli
 > [!NOTE]
 > Lumigator requires an SQL database to hold metadata for datasets and jobs. The local deployment
 > uses SQLite for this purpose.
+>
+> [!NOTE]
+> If you'd like to evaluate against LLM APIs like OpenAI and Mistral, you'll need to have your
+> environment variable set locally for Lumigator pick it up at runtime, or, alternately, inject
+> into the running `backend` docker container.
```

Comment:
I would like a link to `.env.examples` here, just to see the names of the env vars.